spec/models/web_hook_spec.rb

```diff
@@ -64,6 +64,17 @@
 
         WebHook.enqueue_hooks(:post, post_id: 1)
       end
+
+      it 'registers related event on DiscourseEvent' do
```

Comment:
Hmm I think you misunderstood xD.

This tests only ensures that we are registering the event. If the event has been removed, the following specs will still pass but the web hook will no longer be triggered. Ideally, we'll want to have higher level tests that for example, creates a post and ensure that webhook is created.